docs/connectors/getting-started-connectors/README.adoc

```diff
@@ -268,51 +264,54 @@ endif::[]
 [role="_abstract"]
 A *sink* connector consumes messages from a Kafka topic and sends them to an external system.
 
-For this example, you use the *HTTP Sink* connector which consumes the Kafka messages (produced by the source {connectors} instance) and sends the messages to an HTTP endpoint.
+For this example, you use the *HTTP Sink* connector which consumes the Kafka messages (produced by your Data Generator source {connectors} instance) and sends the messages to an HTTP endpoint.
```

Comment:
```suggestion
For this example, you use the _HTTP Sink_ connector which consumes the Kafka messages (produced by your Data Generator source {connectors} instance) and sends the messages to an HTTP endpoint.
```

Reply:
Maybe remove the bold highlighting for the connector name entirely (rather than put it in italics as I'd previously suggested), for consistency with the (non-) highlighting of  "Data Generator source connector" in the previous section?